README.md

```diff
@@ -69,3 +69,27 @@ docker-compose run web pytest --cov-config=.coveragerc --cov=ops_site --cov-repo
 This prototype uses `runserver` as a web server, which is considered insecure
 for production use. This should be replaced with something like `gunicorn` and
 `nginx` before it is deployed beyond prototyping purposes.
+
+## Data model
+
+![OPRE prototype data model](docs/models.png)
+
+To update this visualization, use the django-extensions module to create a new
+dotfile:
```

Comment:
Might want to provide a link explaining what a dotfile is, and/or link to documentation for libraries used

Reply:
Also I'm unclear on the meanings of the linkage lines among the tables. What do the circles at the end of the lines represent? Are join tables / pivot tables represented here?

Reply:
I'll open another PR with some more documentation into the readme because that's smart. I think there's a link to the dotfile documentation in the Python itself but that's not super helpful unless you're reading code. 😛 

FWIW, I also don't know what the circles there are for. They're created by the [graph_models](https://django-extensions.readthedocs.io/en/latest/graph_models.html) command and the documentation doesn't clarify either. So maybe something else for the readme is to mention that so people won't try to read too much into the diagram. Something like:

> This diagram shows all of the tables and which ones have relationships between one another. However, by itself, this diagram should not be used to try to fully understand the model or the nature of the relationships between tables. Instead, for that deeper understanding, this diagram can be used as an aid alongside the Django model code.